app/services/recruitment_cycle_creation_service.rb

```diff
@@ -3,16 +3,19 @@
 class RecruitmentCycleCreationService
   include ServicePattern
 
-  def initialize(year:, application_start_date:, application_end_date:, summary: false)
+  def initialize(year:, application_start_date:, application_end_date:)
     @year = year
     @application_start_date = application_start_date
     @application_end_date = application_end_date
-    @summary = summary
   end
 
   def call
-    RecruitmentCycle.create!(year: @year, application_start_date: @application_start_date, application_end_date: @application_end_date)
-
-    Rails.logger.info { "The new RecruitmentCycle has been successfully created for:\n\nyear: '#{@year}'\napplication_start_date: '#{@application_start_date}'\napplication_end_date: '#{@application_end_date}'\n\n" } if @summary
+    RecruitmentCycle.create!(
+      year: @year,
+      application_start_date: @application_start_date,
+      application_end_date: @application_end_date
+    ).tap do |recruitment_cycle|
+      RolloverJob.perform_later(recruitment_cycle.id)
+    end
```

Comment:
I don't think we need to call `tap` here as we're not chaining it into something else.

```suggestion
    recruitment_cycle = RecruitmentCycle.create!(
      year: @year,
      application_start_date: @application_start_date,
      application_end_date: @application_end_date
    )
    
    RolloverJob.perform_later(recruitment_cycle.id)
```

Reply:
Done